### PR TITLE
chore: IOT project added

### LINKED ISSUE #1850: chore: project addition by Deepak
### Add a new project to the list

Add IOT project of Making the Real time temperature and humidity detection using ESP32 and DHT11.

Screenshot 2023-05-04 033139

### Record

- [X] I have checked the existing [issues](https://github.com/priyankarpal/ProjectsHut/issues)
- [X] I have read the [Contributing Guidelines](https://github.com/priyankarpal/ProjectsHut/blob/main/contributing.md)
- [X] I agree to follow this project's [Code of Conduct](https://github.com/priyankarpal/ProjectsHut/blob/main/CODE_OF_CONDUCT.md)
- [X] I want to work on this issue

### PR DESCRIPTION
**Pull request by k-deepak04**
## Screenshots
image

### CODE REVIEW

File: DB/projects.json [hunk 1]
```diff
@@ -4283,5 +4283,20 @@
...
+    "gitHub": "https://github.com/k-deepak04",
+    "LinkedIn": "https://www.linkedin.com/in/k-deepak04/",
+    "Twitter": "https://twitter.com/k_deepak04"
+  },
+  "Projects": [
+    {
+      "link": "https://github.com/k-deepak04/DHT11RealTime",
+      "title": "IOT ESP32",
+      "description": "Building Real-time Temperature & Humidity sensor using ESP32 & DHT11",
+      "tech": ["C++", "Arduino IDE"]
```
[the remark this answers is not in the document]
Comment (priyankarpal) [reply DB/projects.json:4298]: @k-deepak04 please check these

[the remark this answers is not in the document]
Comment (k-deepak04) [reply DB/projects.json:4298]: okay i will do

[the remark this answers is not in the document]
Comment (k-deepak04) [reply DB/projects.json]: Done plz check

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Comment by priyankarpal** [comment]
Hey @k-deepak04! 👋

For secure collaboration, remember to check out GitHub's commit signature verification feature. It helps ensure the authenticity of commits and prevents tampering.

Learn how it works: [GitHub Docs](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification)

Happy coding! 😄

**Pull request merged**